setup.py

```diff
@@ -38,7 +38,7 @@ def find_version(*file_paths):
     include_package_data=True,
     zip_safe=True,
     python_requires=">=3.6",
-    install_requires=["cloudformation-cli>=0.1,<0.2", "docker>=3.7,<3.8"],
+    install_requires=["cloudformation-cli>=0.1,<0.2", "docker>=3.7,<=4.3.1"],
```

Comment:
This [seems like the latest docker version](https://pypi.org/project/docker/) anyway. Could we just leave the maximum version unpinned?
```suggestion
    install_requires=["cloudformation-cli>=0.1,<0.2", "docker>=3.7"],
```

Reply:
maybe something like this if we want to keep it < 5 ?
```suggestion
    install_requires=["cloudformation-cli>=0.1,<0.2", "docker>=3.7,<=4.*"],
```
```suggestion
    install_requires=["cloudformation-cli>=0.1,<0.2", "docker>=3.7,<5"],
```